pkg/startup/v7/data/master/etc/mdsd.d/mdsd.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MonitoringManagement version="1.0" namespace="{{ .Config.GenevaLoggingNamespace | XMLEscape }}" eventVersion="1" timestamp="2017-08-01T00:00:00.000Z">
+    <Accounts>
+        <Account moniker="{{ .Config.GenevaLoggingAccount | XMLEscape }}" isDefault="true" autoKey="false"/>
```

Comment:
We usually do XML configs based on Derive pattern  https://github.com/openshift/openshift-azure/blob/master/pkg/sync/v7/derive.go#L74

Gives us more control over the file. Is this possible here?

I think you might have double escape/`"`? 
`"{{ .Config.GenevaLoggingAccount | XMLEscape }}"` 
vs `{ .ResourceName | Escape }}` ?

you have `"` in some fields, but not other. What that about?